widget/form.go

```diff
@@ -193,6 +194,20 @@ func (f *Form) checkValidation(err error) {
 	}
 }
 
+func (f *Form) ensureRenderItems() {
+	objects := f.itemGrid.Objects
+	for i, item := range f.Items {
+		if i < len(objects)/2 {
+			continue
+		}
+
+		objects = append(objects, f.createLabel(item.Text))
+		f.setUpValidation(item.Widget, i)
+		objects = append(objects, f.createInput(item))
```

Comment:
Could these appends perhaps be combined into one? It should be slightly faster.

Reply:
Thanks for the pointer, I have updated it to be as close to pre-allocation as possible in latest commit.